prow/images/buildpack-java/README.md

```diff
@@ -0,0 +1,16 @@
+# Buildpack Java Docker Image
+
+## Overview
+
+This folder contains the Buildpack Java image that is based on the Bootstrap image. Use it to build Java components.
+
+The image consists of:
```

Comment:
```suggestion
The Buildpack Java Docker Image contains `java openjdk 8u212-b04`.
```
I figured the build name was not exactly right - please correct me if I'm wrong. Also, bullet points make sense if there is more than one item on the list.